data/brands/shop/variety_store.json

```
@@ -138,6 +138,17 @@
         "shop": "variety_store"
       }
     },
+    {
+      "displayName": "Christmas Tree Shops",
+      "locationSet": {"include": ["us"]},
```

[Overpass turbo](https://overpass-turbo.eu/?w=%22name%22%3D%22Christmas+Tree+Shops%22+global&R) shows that these also are tagged differently, so we also want to match locations tagged as these
```suggestion
      "locationSet": {"include": ["us"]},
      "matchTags": ["shop/department_store", "shop/gift", "shop/houseware", "shop/interior_decoration"],
```